src/transformers/pipelines/image_segmentation.py

```diff
@@ -16,8 +14,13 @@
 
 if is_torch_available():
     import torch
+    from torch.nn import functional as F
```

Comment:
```suggestion
    from torch import nn
```
In other parts of the Transformers library, we prefer to not use single-letter abbreviations. Rather, we use nn.functional in the code.